tdBattlePetScript.toc

```diff
@@ -8,5 +8,6 @@
 ## SavedVariablesPerCharacter: TD_DB_BATTLEPETSCRIPT_BATTLE_CACHE
 ## LoadManagers: AddonLoader
 ## X-LoadOn-Always: delayed
+## X-Curse-Project-ID: 254463
```

Comment:
Add 
```
## X-WoWI-ID: xxx
## X-Wago-ID: xxx
```
once the respective projects have been created. It should then magically pick those up as well.